app/Config/Schema/sessions.php

```diff
@@ -24,9 +24,10 @@
  * cake schema run create Sessions
  *
  */
-class SessionsSchema extends CakeSchema {
+class SessionsSchema extends CakeSchema
```

Comment:
__[staging][code_sniffer]__
* You must use "/**" style comments for a class comment